it/repositories/sifting/SiftingRepositorySpec.scala

```diff
@@ -0,0 +1,88 @@
+package repositories.sifting
+
+import model.EvaluationResults.Green
+import model.Phase3TestProfileExamples.phase3TestWithResult
+import model.ProgressStatuses.PHASE3_TESTS_PASSED
+import model.persisted.{ PassmarkEvaluation, SchemeEvaluationResult }
+import model.{ ApplicationStatus, SchemeId }
+import repositories.onlinetesting.Phase2EvaluationMongoRepositorySpec.phase2TestWithResult
+import repositories.{ CollectionNames, CommonRepository }
+import testkit.MongoRepositorySpec
+
+class SiftingRepositorySpec extends MongoRepositorySpec with CommonRepository {
+
+  override val collectionName = CollectionNames.APPLICATION
+
+  def repository = new SiftingMongoRepository()
+
+  private val userId = "123"
+
+  private val Commercial: SchemeId = SchemeId("Commercial")
+  private val European: SchemeId = SchemeId("European")
+  private val Sdip: SchemeId = SchemeId("Sdip")
+
+  "Sifting repository" should {
+
+    "find eligible candidates" in {
+      createSiftEligibleCandidates(UserId, AppId)
+
+      val candidates = repository.findApplicationsReadyForSifting(Commercial).futureValue
+      candidates.size mustBe 1
+      val candidate = candidates.head
+      candidate.applicationId mustBe Some(AppId)
+    }
+
+    "sift candidate as Passed" in {
+      createSiftEligibleCandidates(UserId, AppId)
+      repository.siftCandidate(AppId, SchemeEvaluationResult(Commercial, "Green")).futureValue
+      val candidates = repository.findApplicationsReadyForSifting(Commercial).futureValue
+      candidates.size mustBe 0
+    }
+
+    /*
+    //    TODO: implement this case
```

Comment:
yes! this is what I mentioned during standup! I noticed it only after raising pull request. Will all TODOs and clear them 👍